.gitlab-ci.yml

```diff
@@ -635,19 +679,46 @@ build/nocuda/openmpi/clang/omp/debug/static:
     FAST_TESTS: "ON"
     BUILD_SHARED_LIBS: "OFF"
 
-build/nocuda/nompi/intel/omp/release/static:
+# nocuda with the oldest supported compiler
+build/nocuda/nompi/intel/core/debug/shared:
   <<: *default_build_with_test
   extends:
     - .quick_test_condition
-    - .use_gko-nocuda-mvapich2-gnu9-llvm8-intel
+    - .use_gko-nocuda-mvapich2-gnu5-llvm39-intel2018
   variables:
     <<: *default_variables
+    # intel with old gcc without include path leads error: identifier "____m128d" is undefined
+    CXX_FLAGS: "-I /opt/intel/include/icc"
     C_COMPILER: "icc"
     CXX_COMPILER: "icpc"
+    BUILD_TYPE: "Debug"
+    FAST_TESTS: "ON"
+
+build/nocuda/nompi/gcc/omp/release/static:
+  <<: *default_build_with_test
+  extends:
+    - .quick_test_condition
+    - .use_gko-nocuda-mvapich2-gnu5-llvm39-intel2018
+  variables:
+    <<: *default_variables
     BUILD_OMP: "ON"
     BUILD_TYPE: "Release"
     BUILD_SHARED_LIBS: "OFF"
 
+build/nocuda-nomixed/nompi/clang/omp/release/static:
```

Comment:
see this

Reply:
because we don't have test for llvm3.9 and we test intel in `build/nocuda/nompi/intel/core/debug/shared`, I change it to clang